tools/download_paks.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+import argparse
+import fnmatch
+import os
+import posixpath
+import spear
+from tqdm import tqdm
+import urllib.request
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--output_dir", required=True)
+    parser.add_argument("--platform", required=True)
+    parser.add_argument("--version_tag", required=True)
+    parser.add_argument("--scene_ids", required=True)
+    args = parser.parse_args()
+
+    assert args.platform in ["Windows", "Linux", "Mac"]
+
+    # create output dir
+    output_dir = os.path.realpath(args.output_dir)
+    os.makedirs(output_dir, exist_ok=True)
+
+    # create a list of available scene_ids
+    available_scene_ids = [ f"kujiale_{str(i).zfill(4)}" for i in range(30) ]
```

Comment:
It would be better to materialize this list explicitly because it is ugly that we handle `warehouse_0000` differently from `kujiale_*`, and we want to be able to skip Kujiale scenes without breaking the logic here.